web/ui/templates/_base.html

```diff
@@ -40,8 +40,9 @@
             <li><a href="{{ pathPrefix }}/alerts">Alerts</a></li>
             <li><a href="{{ pathPrefix }}/graph">Graph</a></li>
             <li><a href="{{ pathPrefix }}/status">Status</a></li>
```

Comment:
Also note that the status page does contain more than "targets" – it contains the _status of each target_.